lib/fog/ovirt/requests/compute/v4/add_volume.rb

```diff
@@ -32,12 +32,12 @@ def add_options_defaults(options)
             options[:disk] ||= {}
             options[:disk][:storage_domains] ||= [client.system_service.storage_domains_service.storage_domain_service(options[:storage_domain_id]).get]
             options[:disk][:provisioned_size] ||= options.delete(:provisioned_size)
-            options[:disk][:type] ||= options.delete(:type)
             options[:disk][:format] ||= options.delete(:format)
-            options[:disk][:sparse] = options.delete(:sparse) unless options[:disk][:sparse].present?
+            options[:disk][:sparse] = options.delete(:sparse) == "true"
+            options[:disk][:wipe_after_delete] = options.delete(:wipe_after_delete)
```

Comment:
I assume `:wipe_after_delete` is a boolean. Shouldn't it be handled the same way as other booleans, like `:sparse` or `:bootable`?